soc/arc/snps_qemu/Kconfig.defconfig

```diff
@@ -27,6 +27,9 @@ config NUM_IRQ_PRIO_LEVELS
 config NUM_IRQS
 	default 25
 
+config ARC_MPU_VER
+	default 2
```

Comment:
Do ARC HS processors can be configured with MPU version 2?

Reply:
actually, it is version 3, but we don't support version 3 currently. and v3 is similar to v2 with minimal 32byte region size.
relative discussion:
https://github.com/zephyrproject-rtos/zephyr/issues/31416
https://github.com/zephyrproject-rtos/zephyr/pull/32573

Reply:
Right, so it worth addition of a comment explaining that. At least in the commit comment. Something like:
> Technically ARC HS supports MPUv3, but not v2. But given MPUv3 is the same as v2 but with minimal region size of 32 bytes, we may assume MPUv3 is just a subset of MPUv2.